### ISSUE TITLE
How to set the number of maximum epoch for CycleGan?

### ISSUE THREAD

**Issue by Zhaoyisong** [junyanz/pytorch-CycleGAN-and-pix2pix#870]
Hello,I'm using my own dataset training CycleGan model,I want to set the number of maximum epoch,but I can't find any parameters to set that.Thx

**Comment by junyanz** [comment]
Please refer to `--niter' and `--niter_decay`. See [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py#L30) for more details.

**Comment by Zhaoyisong** [comment]
--niter and --niter_decay ，The sum of these two figures is 200（default）,these are set learning rate but not epoch number in details.

**Comment by junyanz** [comment]
It sets the number of epochs. During the 1st and `--niter` epochs, it will use the initial learning rate. During the `--niter` and `--niter`+`--niter_decay`, it will use a decaying learning rate.

**Comment by niannianyouyu** [comment]
You can set the number of epochs by using the flag `--n_epochs 300` and `--n_epochs_decay 200`(300 epochs for initial learning rate and 200 epochs for a decaying learning rate), and there will be 500 epochs of training .
